### PR TITLE
Remove SECURITY.md file and point to fairlearn-internal mailing list

### LINKED ISSUE #290: SECURITY.md file
It seems the `SECURITY.md` file is from the time where `fairlearn` was under the _microsoft_ org. Now that it's not, does it need to be there?

### PR DESCRIPTION
**Pull request by romanlutz**
closes #290 
related to #433 

Signed-off-by: Roman Lutz <rolutz@microsoft.com>

### CODE REVIEW

**Comment by adrinjalali** [comment]
There are `FutureWarnings` there too. I suggest opening a separate PR to fix those issues first and then getting this one in. We should have crone scheduled CI to test against these new releases of our dependencies. cc @riedgar-ms

File: fairlearn/reductions/__init__.py [hunk 1]
```diff
@@ -18,15 +18,7 @@
 from ._moments import BoundedGroupLoss, SquareLoss, ZeroOneLoss  # noqa: F401
 from ._moments import ClassificationMoment, LossMoment  # noqa: F401
 
-_exponentiated_gradient = [
```
Comment (riedgar-ms) [inline fairlearn/reductions/__init__.py:21]: This seems.... unrelated?

Comment (romanlutz) [reply fairlearn/reductions/__init__.py:21]: Flake8 is requiring this to change. RST902 failure and I can't override it with `noqa` it seems. So I played around with it and this is the only solution I can come up with. This is probably hitting every PR right now (?)

**Comment by romanlutz** [comment]
> There are `FutureWarnings` there too. I suggest opening a separate PR to fix those issues first and then getting this one in. We should have crone scheduled CI to test against these new releases of our dependencies. cc @riedgar-ms

Fair enough. I'll create another PR for the `flake8` fixes. Where would I find the `FutureWarnings`?

File: fairlearn/reductions/__init__.py (hunk 1 above)
Comment (romanlutz) [reply fairlearn/reductions/__init__.py:21]: Moving this a different PR

**Review by adrinjalali — APPROVED** [review]

**Review by MiroDudik — APPROVED** [review]

**Pull request merged**